### PR TITLE
NO-ISSUE: Changes for running flightctl server and agent VM locally

### PR DESCRIPTION
**Pull request by vraiti**
Changed some of the YAML indentation to make helm happy, updated the documentation to reflect that the VM will have a login of user/user, and added CLI option to agent VM to emulate TPM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ this only works on a Linux host.
 ```
 # will create the cluster, and the agent config files in bin/agent which will be embedded in the image
 make deploy
-make agent-vm agent-vm-console # user/password is redhat/redhat
+make agent-vm agent-vm-console # user/password is user/user
```
Comment (hexfusion) [inline README.md:61]: :heart:

**Review by hexfusion — COMMENTED** [review]
Thanks for the PR! You need to prefix all commits with `NO-ISSUE:` as well as sign them. :pray:

File: deploy/agent-vm.mk [hunk 2]
```diff
@@ -10,14 +10,14 @@ agent-vm: bin/output/qcow2/disk.qcow2
 	sudo cp bin/output/qcow2/disk.qcow2 $(VMDISK)
 	sudo chown libvirt:libvirt $(VMDISK) 2>/dev/null || true
 	sudo virt-install --name $(VMNAME) \
+					  --tpm backend.type=emulator,backend.version=2.0,model=tpm-tis \
```
Comment (mangelajo) [inline deploy/agent-vm.mk:13]: 🚀

**Comment by mangelajo** [comment]
Yeah, as @hexfusion you need to edit your commit `git commit --amend -S` and put NO-ISSUE:  (including the ':') in front of the commit title.

Thanks @Vance-Raiti !

**Pull request closed without merging**